app/javascript/components/editor/TestRunOutput.tsx

```diff
@@ -11,7 +11,7 @@ export const TestRunOutput = ({
   const hasTasks =
     testRun.version >= 3 &&
     testRun.tasks.length > 0 &&
-    testRun.tests.filter((t) => t.taskId == null || t.taskId == undefined) == 0
+    testRun.tests.every((t) => t.taskId)
```

Comment:
Just to be sure, can taskId validly be 0 (ie do tracks every index from 0)? Just because 0 is falsey in JS.

Reply:
Output `0` as the task ID would be an error in the test runner, but I've made it more explicit